### PR TITLE
LPD-36485 Improve description

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts [hunk 1]
```diff
@@ -434,7 +434,6 @@ export class PageEditorPage {
 	async editHTMLEditable({
 		editableId,
 		fragmentId,
-		useBackwardCompatibility = false,
 		value,
```
Comment (ealonso) [inline modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts:437]: @victorg1991 I think you need to update methods calls of editHTMLEditable(

Comment (victorg1991) [reply modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts:437]: :thinking:

Comment (victorg1991) [reply modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts:437]: true

**Comment by victorg1991** [comment]
ci:test:page-management-playwright

**Comment by ealonso** [comment]
ci:test:page-management-playwright

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**